datagrepper/app.py

```diff
@@ -578,7 +547,8 @@ def make_charts(chart_type):
     start = flask.request.args.get("start", None)
     end = flask.request.args.get("end", None)
     delta = flask.request.args.get("delta", None)
-    start, end, delta = assemble_timerange(start, end, delta)
+    with app.app_context():
```

Comment:
Here again, the app context is automatically created by Flask, we shouldn't touch it here, but we can do this in the unit tests.